css-grid-1/Overview.bs

```diff
@@ -1876,45 +1875,6 @@ Flexible Lengths: the ''fr'' unit</h4>
 	Each column or row's share of the <a>leftover space</a> can be computed as the column or row's
 	<code>&lt;flex> * &lt;leftover space> / &lt;sum of all <a>flex factors</a>&gt;</code>.
```

Comment:
I think `&lt;flex>` should be `&lt;fraction>`

Reply:
Right. Done.